From: Sarah Sharp <sarah.a.sharp@linux.intel.com>
Date: Tue, 1 Dec 2009 10:37:07 -0800
Subject: [stable] USB: xhci: Add correct email and files to MAINTAINERS entry.
Bugzilla: 555084

(Pulled from -stable series)

commit 36d0344c254a7b333272757f858c403ea3a2d29f upstream.

Add the xHCI driver files to its MAINTAINERS entry so that I'm Cc'd on
cleanup patches.  Update the email address to one I actually use for
sending patches and responding to Linux mailing list emails.

Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

Signed-off-by: Aristeu Rozanski <aris@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 39e61d3..20017ed 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -5603,9 +5603,11 @@ S:	Maintained
 F:	drivers/net/wireless/rndis_wlan.c
 
 USB XHCI DRIVER
-M:	Sarah Sharp <sarah.a.sharp@intel.com>
+M:	Sarah Sharp <sarah.a.sharp@linux.intel.com>
 L:	linux-usb@vger.kernel.org
 S:	Supported
+F:	drivers/usb/host/xhci*
+F:	drivers/usb/host/pci-quirks*
 
 USB ZC0301 DRIVER
 M:	Luca Risolia <luca.risolia@studio.unibo.it>
